crates/sui-graphql-rpc/src/types/address.rs

```diff
@@ -106,6 +106,10 @@ impl Address {
             .extend()
     }
 
+    /// The coin objects for the given address.
+    ///
```

Comment:
Nit whitespace

Reply:
That's on purpose such that it is displayed on a newline
<img width="861" alt="image" src="https://github.com/MystenLabs/sui/assets/135084671/e19d675b-68f3-4b9f-a88f-04ea702147c6">